.travis.yml

```diff
@@ -7,7 +7,7 @@ cache:
 before_script:
   - chmod +x scripts/docs
 script:
-  - npm run cover
+  - npm run cover && npm run lint
```

Comment:
Sweet! I didn't know an extra script line would still be run if the previous script failed. Best of both experiences.